Services/JobRunner/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿
+// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Collections.Generic;
+using System.Composition.Hosting;
+
+namespace JobRunner
+{
+    public static class ServiceCollectionExtensions
```

Comment:
A variant of this class exists in flow.management service. We should reuse code and avoid dup